### PR TITLE
Bugfix web backup button

### PR DESCRIPTION
**Pull request by Thykof**

### CODE REVIEW

File: wails-frontend/src/pages/passwordPrompt.tsx [hunk 1]
```diff
@@ -48,14 +48,11 @@ function PasswordPrompt() {
...
-    const { password } = form;
-
-    // reset error state
-    setError(null);
+    const formObject = parseForm(e);
+    const { password } = formObject;
 
-    if (!hasMoreThanFiveChars(password)) {
-      setError({ password: 'Password must have at least 5 characters' });
+    if (!password || !password.length) {
```
Comment (mazmassa) [inline wails-frontend/src/pages/passwordPrompt.tsx:54]: Question: do we need both!?

**Review by peterjah — APPROVED** [review]

**Review by mazmassa — APPROVED** [review]
LGTM!

File: wails-frontend/src/pages/passwordPrompt.tsx (hunk 1 above)
Comment (mazmassa) [reply wails-frontend/src/pages/passwordPrompt.tsx:54]: Feel free to close if is needed!

Comment (Thykof) [reply wails-frontend/src/pages/passwordPrompt.tsx:54]: variable returned by parseForm is type any, so yes

**Pull request merged**